muikku/src/main/webapp/resources/theme-muikku/sass/flex/evaluation-index.scss

```diff
@@ -409,105 +483,80 @@ body {
   
 }
 
-// SEARCH WRAPPER 
-.evaluation-search-wrapper {
-  background: $evaluation-search-wrapper-background-color;
+// FILTER WRAPPER
+.evaluation-filters-wrapper {
+  background: $evaluation-filter-wrapper-background-color;
   bottom: 0;
-  box-shadow: -5px 5px 20px $evaluation-search-wrapper-background-boxshadow-color;
+  box-shadow: -10px 0 10px $evaluation-filter-wrapper-background-boxshadow-color;
   display: none;
   padding: 0;
   position: fixed;
   right: calc(0px - 360px);
   top: 60px;
   width: 360px;
-  z-index:20;
+  z-index: 20;
   
   h3 {
     color: $body-font-color;
     font-size: 18px;
     font-weight: 300;
+    margin: 10px;
     text-transform: uppercase;
   }
   
 }
 
-.evaluation-search-container {
-  padding: 10px;
-  
-  .evaluation-search {
-    @include flexbox;
-    box-sizing: border-box;
-    flex-flow: row nowrap;
-  }
+// WORKSPACES WRAPPER
+.evaluation-workspaces-wrapper {
+  background: $evaluation-workspaces-wrapper-background-color;
+  bottom: 0;
+  box-shadow: 10px 0 10px $evaluation-workspaces-wrapper-background-boxshadow-color;
+  display: none;
+  left: calc(0px - 360px);
```

Comment:
Hound is wrong in this occasion, calc does not even work if we use 0 without units! Go bark to someone else you stupid dog!